cashier-paddle.md

```diff
@@ -94,7 +94,7 @@ Before using Cashier, you must add the `Billable` trait to your model definition
         use Billable;
     }
 
-The trait can be implemented on other types of models in the same app as well:
+If you have billable entities that are not users, you may add the trait to those classes:
```

Comment:
Think the following keyword indicates more that the trait can be used in combination with other models

```suggestion
If you have billable entities that are not users, you may also add the trait to those classes:
```